### PR TITLE
main/dbus: start user service in $HOME and system service in /

### PR DESCRIPTION
**Pull request by triallax**
e.g. so that yakuake, when added to kde's autostarted apps, starts in $HOME and not in /etc/dinit.d/user

### CODE REVIEW

File: main/dbus/files/dbus.user [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # dbus session bus user service
 
-type               = process
-command            = /usr/libexec/dbus-session.wrapper --print-address=4
+type = process
+command = /usr/libexec/dbus-session.wrapper --print-address=4
 ready-notification = pipefd:4
+# So dbus services don't start in /etc/dinit.d/user :')
+working-dir = $HOME
```
Comment (triallax) [inline main/dbus/files/dbus.user:7]: is this supposed to be left unquoted? i'm not sure whether dinit does the weird shell shenanigans

Comment (nekopsykose) [reply main/dbus/files/dbus.user:7]: it doesn't p sure

File: main/dbus/files/dbus.user [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # dbus session bus user service
 
-type               = process
-command            = /usr/libexec/dbus-session.wrapper --print-address=4
+type = process
+command = /usr/libexec/dbus-session.wrapper --print-address=4
 ready-notification = pipefd:4
+# So dbus services don't start in /etc/dinit.d/user :')
```
Comment (q66) [inline main/dbus/files/dbus.user:6]: don't really need the comment

**Comment by q66** [comment]
i'd also modify the system bus service while at it and also have it start in a better location (probably `/`)

**Pull request merged**